### DISCUSSION TITLE
Tauri mobile support

### DISCUSSION THREAD

**Discussion by danielpetrica** [NativePHP/laravel#433] [General]
Hi, I don't know how Tauri will be implemented but I saw that they are implementing mobile support inside Tauri. do you think it's possible in the future to bring this to NativePhp too or are we limited by PHP binaries?

**Comment by JWprogrammer** [comment]
Hi. I checked, PHP binaries work fine with ARM architecture on regular android mobile devices. I just installed the Termux console application and installed PHP from the package manager

**Comment by JWprogrammer** [comment]
![изображение](https://github.com/user-attachments/assets/947ccb0b-e782-406e-a062-1ab2cf2c53c0)

**Comment by simonhamp** [comment]
It's kind of unlikely that we'll do this via Tauri as we now have our own first-party implementation in [NativePHP for mobile](https://nativephp.com/mobile)
